apps/user-profile/src/app/pages/profile-page/profile-page.component.html

```diff
@@ -22,8 +22,11 @@
 
         <div *ngSwitchCase="'preferredMail'">
           <a class="center-self" href="mailto:{{email}}">{{email}}</a>
-          <button (click)="changeEmail()" class="ml-2"
-                  mat-stroked-button>{{'PROFILE_PAGE.CHANGE_EMAIL'|customTranslate | translate}}</button>
```

Comment:
Please also remove unused label `PROFILE_PAGE.CHANGE_EMAIL` from `en.json` and `cs.json`. Afterwards this PR can be merged.